{{cookiecutter.app_name}}/Makefile

```diff
@@ -1,19 +1,48 @@
-.PHONY: env-setup env-teardown dev install-dependencies test
+include .env
+ifdef ENV
+include .env.$(ENV)
+endif
+
+.PHONY: env-setup env-teardown db/migrate db/rollback migratoin/create migration/status dev install-dependencies test wait-for-postgres
```

Comment:
A minor typo
```suggestion
.PHONY: env-setup env-teardown db/migrate db/rollback migration/create migration/status dev install-dependencies test wait-for-postgres
```

Reply:
Good catch. Fixed in 3a3eebf990a6f210eafa8e9255ea09b03f63b8a7